our-brothers-web/src/app/shared/components/registration-progress-bar/registration-progress-bar.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="progressbar-container">
+<div class="progressbar-container container">
```

Comment:
`container progressbar-container`

Reply:
Usually, the common class comes before the specific one

Reply:
fix